src/sdk/model/model.ts

```diff
@@ -39,11 +39,19 @@ export type Output = {
     readonly source: string
 };
 
+export type CodeSnippetContext = {
+    readonly language: string;
```

Comment:
Should we use the `Language` type?

Reply:
I am thinking to test having context with different type of files, for example when working with `.jsx` file, it will be also nice to include `.css`'s context, so I would like to leave it to plain text to have more flexibility for doing experiments